src/main/java/com/jcabi/dynamodb/maven/plugin/Instances.java

```diff
@@ -51,6 +51,9 @@
  * @since 0.1
  * @checkstyle ClassDataAbstractionCoupling (500 lines)
  * @see <a href="http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Tools.html">DynamoDB Local</a>
+ * @todo #44:30min Let's avoid code duplication between
```

Comment:
@dmzaytsev It's 44 because it's the next issue to be created by the puzzle. The issue 42 already exists!